api/openapi-spec/swagger.json

```diff
@@ -2246,7 +2298,29 @@
           "type": "string"
         },
         "name": {
-          "description": "name is the name of the resource in question.",
+          "description": "name is the name of the resource in question.\n\nPossible enum values:\n - `configmaps`: , number\n - `cpu`: CPU, in cores. (500m = .5 cores)\n - `ephemeral-storage`: Local ephemeral storage, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024) The resource name for ResourceEphemeralStorage is alpha and it can change across releases.\n - `limits.cpu`: CPU limit, in cores. (500m = .5 cores)\n - `limits.ephemeral-storage`: Local ephemeral storage limit, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)\n - `limits.memory`: Memory limit, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)\n - `memory`: Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)\n - `persistentvolumeclaims`: , number\n - `pods`: Pods, number\n - `replicationcontrollers`: ReplicationControllers, number\n - `requests.cpu`: CPU request, in cores. (500m = .5 cores)\n - `requests.ephemeral-storage`: Local ephemeral storage request, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)\n - `requests.memory`: Memory request, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)\n - `requests.storage`: Storage request, in bytes\n - `resourcequotas`: , number\n - `secrets`: , number\n - `services.loadbalancers`: , number\n - `services.nodeports`: , number\n - `services`: Services, number\n - `storage`: Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)",
```

Comment:
metrics are quite a pain. I will double check this part or ignore these.

Reply:
Yeah, these comments are messy. There's a possibility we just go there and fix these.